command/untaint_test.go

```diff
@@ -0,0 +1,478 @@
+package command
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/mitchellh/cli"
+)
+
+func TestUntaint(t *testing.T) {
```

Comment:
I don't think this is a noop - it's the "basic" use case - `test_instance.foo` starts tainted and ends up untainted.